index.html

```diff
@@ -159,6 +159,27 @@
 <span class="dl2">x</span><input type="number" pattern="\d+(\.\d*)?" id="e0" value="0" oninput="update(4)"/>
 <span class="dl2">y</span><input type="number" pattern="\d+(\.\d*)?" id="e1" value="0" oninput="update(4)"/>
 <span class="dl2">z</span><input type="number" pattern="\d+(\.\d*)?" id="e2" value="0" oninput="update(4)"/>
+<p class="pi" id="ip5">Triad of points, <b>P</b><sub>1</sub>, <b>P</b><sub>2</sub>, <b>P</b><sub>3</sub>, 
```

Comment:
Ok, this new type of conversion seems useful.
I only disagree about the notation and naming. Among other things, the book does not say "triad".
Can we call the title "Coordinate system from three points" and the explanation "Points O, P_X, P_Y define a coordinate system with origin O, P_X on the x-axis, and P_Y on the x-y-plane."?
The only thing that strikes me is the origin is mostly irrelevant to the calculation, maybe it should be left out?